lib/rules/availableRules.js

```diff
@@ -10,25 +10,25 @@ const rules = {
     },
   }),
   maxChar: (input, config) => ({
-    message: () => `The commit message is not allowed to be longer as ${config.rules['max-char']} character, but is ${input.length} character long. Consider writing a body.`,
+    message: () => `The commit message is not allowed to be longer as ${config.rules['max-char']} character, but is ${input.trim().length} character long. Consider writing a body.`,
```

Comment:
Would be better if we trim them here: https://github.com/JPeer264/node-semantic-git-commit-cli/blob/master/lib/questions.js#L62

Then it would apply to every rule automatically. So we do not have to touch this file.